app/assets/stylesheets/utilities/layout.scss

```diff
@@ -13,3 +13,11 @@
 .inline-block {
   display: inline-block;
 }
+
+.dashboard-colour {
+  background:#2b8cc4;
```

Comment:
It'd be good to use the standard `govuk-colour` here, I assume they've got a fitting blue?

[Here's an example](https://github.com/alphagov/govwifi-admin/blob/master/app/assets/stylesheets/components/label_inside_input.scss#L3) from another stylesheet in the application, and the [list of colours can be found on the design system site](https://design-system.service.gov.uk/styles/colour/).